crates/ty_python_semantic/src/types/tuple.rs

```diff
@@ -370,12 +374,12 @@ impl<T> FixedLengthTuple<T> {
         self.0
     }
 
-    pub(crate) fn elements(&self) -> impl DoubleEndedIterator<Item = &T> + ExactSizeIterator + '_ {
-        self.0.iter()
+    pub(crate) fn elements(&self) -> &[T] {
+        &self.0
     }
 
-    pub(crate) fn all_elements(&self) -> impl Iterator<Item = &T> {
-        self.0.iter()
+    pub(crate) fn all_elements(&self) -> &[T] {
+        &self.0
     }
```

Comment:
Why do we have both of these methods? 😆 (I realize it pre-dates this PR and you probably don't know; it's just funny)

Reply:
I don't know 😅 I deleted `elements` so that it's aligned with the variable length tuple